src/dbsync/tests/utils/pipelineNodes_test.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
```

Comment:
Please, use: 
#ifndef _PIPELINE_NODES_TESTS_H
#define _PIPELINE_NODES_TESTS_H
...
#endif // _PIPELINE_NODES_TESTS_H

instead of #pragma once.